src/halmos/__main__.py

```diff
@@ -221,41 +223,29 @@ def mk_calldata(
     calldata.create(fun_abi, cd)
 
 
-def mk_callvalue() -> Word:
-    return BitVec("msg_value", 256)
-
-
-def mk_balance() -> Word:
-    return Array("balance_0", BitVecSort(160), BitVecSort(256))
-
-
 def mk_block() -> Block:
+    # foundry default values
     block = Block(
-        basefee=ZeroExt(160, BitVec("block_basefee", 96)),  # practical limit 96 bit
-        chainid=ZeroExt(192, BitVec("block_chainid", 64)),  # chainid 64 bit
-        coinbase=mk_addr("block_coinbase"),  # address 160 bit
-        difficulty=BitVec("block_difficulty", 256),
-        gaslimit=ZeroExt(160, BitVec("block_gaslimit", 96)),  # practical limit 96 bit
-        number=ZeroExt(192, BitVec("block_number", 64)),  # practical limit 64 bit
-        timestamp=ZeroExt(192, BitVec("block_timestamp", 64)),  # practical limit 64 bit
+        basefee=ZERO,
+        chainid=con(31337),
+        coinbase=address(0),
+        difficulty=ZERO,
+        gaslimit=con(2**63 - 1),
+        number=ONE,
+        timestamp=ONE,
     )
-    block.chainid = con(1)  # for ethereum
     return block
 
 
 def mk_addr(name: str) -> Address:
     return BitVec(name, 160)
 
 
-def mk_caller(args: HalmosConfig) -> Address:
-    return mk_addr("msg_sender") if args.symbolic_msg_sender else magic_address
-
-
 def mk_this() -> Address:
     # NOTE: Do NOT remove the `con_addr()` wrapper.
     #       The return type should be BitVecSort(160) as it is used as a key for ex.code.
     #       The keys of ex.code are compared using structural equality with other BitVecRef addresses.
-    return con_addr(magic_address + 1)
+    return con_addr(FOUNDRY_TEST)
```

Comment:
might be nice to standardize on `address()` rather than `con_addr()`, wdyt?

Reply:
good idea. they are slightly different though right now--value range check only in con_addr. perhaps it would be better to add the value range check in the con(), and then replace con_addr() with address(). but we need to make sure this doesn't break others. let's work on it in a separate pr.